packages/app-config-writer/src/prompt/navigation-config.ts

```diff
@@ -44,9 +44,10 @@ export async function promptInboundNavigationConfig(
  * @param input the text input to validate
  * @param inputName the name of the input as seen by the user
  * @param maxLength optional, the maximum length of text to allow
+ * @param applyRegex optional, not all fields need to be validated with regex
  * @returns true, if all validation checks pass or a message explaining the validation failure
  */
-function validateText(input: string, inputName: string, maxLength = 0): boolean | string {
+export function validateText(input: string, inputName: string, maxLength = 0, applyRegex = true): boolean | string {
```

Comment:
Fairly obscure pararm `applyRegex`. What about `disallowSpecialCharacters` ?

Reply:
I've applied a change that takes a list of allowed characters which is appended to the alphanumeric regex so that any other character is handled.